.github/pull_request_template.md

```diff
@@ -6,6 +6,19 @@ Fixes #X.  <!-- Insert issue number here if applicable. -->
 
 ## To Do
 
-- [ ] Documentation. (If you've add a new command-line flag, for example, find the appropriate page under `docs/` to describe it.)
-- [ ] Changelog. (Add an entry to `docs/changelog.rst` near the top of the document.)
-- [ ] Tests. (Encouraged but not strictly required.)
+<!--
+- If you believe one of below checkpoints is not required for the change you
+  are submitting, cross it out and check the box nonetheless to let us know.
+  For example: - [x] ~Changelog~
+- Please remove the descriptive sentences in braces from the enumeration below,
+  which helps to unclutter your PR description.
+- Regarding the changelog, often it makes sense to add your entry only once
+  reviewing is finished. That way you might prevent conflicts from other PR's in
+  that file, as well as keep the chance high your description fits with what
+  the latest revision of your feature/fix.
+- Regarding documentation, bugfixes often don't require additions to the docs.
+-->
+
+- [ ] Documentation. (If you've added a new command-line flag, for example, find the appropriate page under `docs/` to describe it.)
+- [ ] Changelog. (Add an entry to `docs/changelog.rst` to the bottom of one of the lists near the top of the document.)
+- [ ] Tests. (Very much encouraged but not strictly required.)
```

Comment:
Please @RollingStar prove-read my changes to the PR template. Does it address the things you mentioned properly. Thanks!

Reply:
Looks good!